components/raftstore/src/store/peer.rs

```diff
@@ -925,6 +925,14 @@ where
     /// this peer has raft log gaps and whether should be marked busy on
     /// apply.
     pub last_leader_committed_idx: Option<u64>,
+
+    /// Used to record uncampaigned regions, which are the new regions
+    /// created when a follower applies a split. If the follower becomes a
+    /// leader, a campaign is triggered for those regions.
+    /// The first element is the region id, the second element is the time when
+    /// pending regions are added, used to clear the pending regions after an
+    /// election timeout.
+    pub uncampaigned_new_regions: (Vec<u64>, Instant),
```

Comment:
```suggestion
    pub uncampaigned_new_regions: Option<(Vec<u64>, Instant)>,
```

nit: Maybe it can be an option, as a peer does not always have `uncampaigned_new_regions`.